src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService useService = new UserService();
+        User user = useService.findByEmail(email);
+
+        if (user == null) {
+            return false;
+        }
+
+        if (!user.getPassword().equals(password)) {
+            return false;
+        }
+        return true;
```

Comment:
if structure usually not required when we aim to return boolean value
```suggestion
        return user != null && user.getPassword().equals(password);
```